From b6d6499c598caaa50f492c6558b649fcd608a1f6 Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Sat, 30 Jan 2016 17:19:03 -0700
Subject: [PATCH 2/3] Revert "close stdin before running install scripts"

This reverts commit e374e6829cea3512f0b4a4069c5a6168f0f8d8a0.

Arch Linux ARM packages have use cases for this feature, such as
prompting the user to flash a new kernel to a bare partition. Removing
this feature will undoubtedly cause more problems than it intends to
solve.

Signed-off-by: Kevin Mihelich <kevin@archlinuxarm.org>
---
 lib/libalpm/util.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/libalpm/util.c b/lib/libalpm/util.c
index 4a4847d..5e74462 100644
--- a/lib/libalpm/util.c
+++ b/lib/libalpm/util.c
@@ -602,7 +602,6 @@ int _alpm_run_chroot(alpm_handle_t *handle, const char *cmd, char *const argv[],
 
 	if(pid == 0) {
 		/* this code runs for the child only (the actual chroot/exec) */
-		close(0);
 		close(1);
 		close(2);
 		while(dup2(child2parent_pipefd[1], 1) == -1 && errno == EINTR);
-- 
2.7.1

